src/util.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
             typeof console.error === 'function') {
             console.error(_);
         }
+    },
+    warn: function(_) {
+        if (typeof console === 'object' && 
+            typeof console.warn === 'function') {
+            console.warn(_);
+        }
```

Comment:
The context this is being used may cause hundreds of duplicate warnings to be printed — maybe worth adopting something like `warnOnce` [from GL JS](https://github.com/mapbox/mapbox-gl-js/blob/master/src/util/util.js#L318-L332)?

Reply:
Great point, implemented in https://github.com/mapbox/mapbox.js/pull/1317/commits/5a25f6241c21be672d67d2859bdfe0aeec3f0b0d